templates/search/indexes/api/item_text.txt

```diff
@@ -1,4 +1,5 @@
 {{object.title}}
 {{object.description}}
 {{object.collection__title}}
-{{object.created_by__full_name}}
\ No newline at end of file
+{{object.created_by.full_name}}
```

Comment:
is it really `__` in some cases and `.` in others?

Reply:
I think it's `.` in templates while its `__` in the kwarg. I looked in the index after reindexing and the names are there.

Reply:
so L3 should be changed to the same?